curves/src/templates/short_weierstrass_jacobian/affine.rs

```diff
@@ -180,6 +188,16 @@ impl<P: Parameters> AffineCurve for Affine<P> {
     }
 }
 
+impl<P: Parameters> ToMinimalBitRepresentation for Affine<P> {
+    fn to_minimal_bit_representation(&self) -> Vec<bool> {
+        let mut res_bits = self.x.to_bits_le();
+        res_bits.push(self.y.to_bits_le().first().unwrap().clone());
+        res_bits.push(self.infinity);
```

Comment:
For SW, we get the (1) full x, (2) the LSB of y (i.e., odd or even), and (3) the infinity bit.

We will see later that in TE, we only need the full x.

Reply:
Does the bit decomposition still pad with extra bits?

Reply:
bit decomposition of the field elements*

That, is is `bls12_377::Fq.to_bites` 377 bits or 384 bits?

Reply:
Now it is 377. There is an additional step in Fp256/384/... to truncate the last a few bits for `to_bits_le`. And for `to_bits_be`, it now simply uses the result of `to_bits_le` and reverse it.